### PR TITLE
Route to get a conversation

### PR DESCRIPTION
**Pull request by philipperolet**

### CODE REVIEW

File: front/pages/api/w/[wId]/assistant/conversations/[cid]/index.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    return apiError(req, res, {
+      status_code: 403,
+      api_error: {
+        type: "workspace_auth_error",
+        message:
+          "Only users of the current workspace can access chat sessions.",
+      },
+    });
+  }
+  if (!(typeof req.query.cId === "string")) {
```
Comment (philipperolet) [inline front/pages/api/w/[wId]/assistant/conversations/[cid]/index.ts:62]: Or do we want to also io-tsize req.query?

Comment (spolu) [reply front/pages/api/w/[wId]/assistant/conversations/[cid]/index.ts:62]: Yes io-ts everywhere

Comment (spolu) [reply front/pages/api/w/[wId]/assistant/conversations/[cid]/index.ts:62]: Ah no sorry this is rien but we want to test it’s a strong

Comment (spolu) [reply front/pages/api/w/[wId]/assistant/conversations/[cid]/index.ts:62]: String

Comment (spolu) [reply front/pages/api/w/[wId]/assistant/conversations/[cid]/index.ts:62]: So fine 👍

File: front/pages/api/w/[wId]/assistant/conversations/[cid]/index.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      },
+    });
+  }
+  const conversationId = req.query.cid;
+
+  switch (req.method) {
+    case "GET":
+      const conversation = await getConversation(
+        auth,
+        conversationId as string
```
Comment (spolu) [inline front/pages/api/w/[wId]/assistant/conversations/[cid]/index.ts:77]: No as you can ensure it’s a string here by setting it to query and testing it’s not null etc

**Review by spolu — COMMENTED** [review]
Remove as and otherwise LGTM

**Review by spolu — APPROVED** [review]

**Pull request merged**